### DISCUSSION TITLE
tfly messages adjustments in depth

### DISCUSSION THREAD

**Discussion by Truongvv10** [Zrips/CMI#7231] [Enhancement]
An option to make tfly display _```messages in different ways```_, like:
- Actionbar
- Bossbar
- Chat
- Title
- Goal

An option to have tfly _```messages disabled```_ would be nice so we can stop the spam messages it gets in actionbar. This gets annoying when using for example towny where actionbar messages gets displayed repeatedly where both just go back and forth flickering.

An option to have tfly _```world blacklist/white```_, so players can't fly in certain worlds.

An option to have tfly be _```silent output```_, both for players and console. Probably with /cmi tfly Zrips +30 -s

**Comment by mrfloris** [comment]
Feature enhancements (suggestions) are supposed to be posted here: https://github.com/Zrips/CMI/discussions

I wouldn't mind having a different way of displaying tfly information. btw.

**Comment by Truongvv10** [comment]
> Feature enhancements (suggestions) are supposed to be posted here: https://github.com/Zrips/CMI/discussions
> 
> I wouldn't mind having a different way of displaying tfly information. btw.

My bad, will do feature request at appropriate place
